### PR TITLE
docs: delete missing part path

### PR DESCRIPTION
**Pull request by devthejohn**
Hi I think this part is not existing in the repo right now.

### CODE REVIEW

File: docs/repo/layout.md [hunk 1]
```diff
@@ -168,7 +168,6 @@ Small utility crates.
 
 - [`tasks`](../../crates/tasks): An executor-agnostic task abstraction, used to spawn tasks on different async executors. Supports blocking tasks and handles panics gracefully. A tokio implementation is provided by default.
 - [`metrics/common`](../../crates/metrics/src/common): Common metrics types (e.g. metered channels)
```
Comment (Rjected) [inline docs/repo/layout.md:170]:
```suggestion
- [`metrics/common`](../../crates/metrics/src/common): Common metrics types (e.g. metered channels)
- [`metrics/metrics-derive`](https://github.com/rkrasiuk/metrics-derive): A derive-style API for creating metrics
```

**Review by Rjected — CHANGES_REQUESTED** [review]
should be this, we extracted the crate into a separate repo because it is generally useful

File: docs/repo/layout.md (hunk 1 above)
Comment (devthejohn) [reply docs/repo/layout.md:170]: Aha nice, It take me like 40mins to search around, sorry I think it not exist.

**Review by Rjected — APPROVED** [review]
ty!

**Pull request merged**